_assets/js/utils/createRange.js

```diff
@@ -0,0 +1,8 @@
+const createRange = (start, end, step) => {
```

Comment:
Do we need es6 and js files?

Reply:
Actually, it looks like some of the files in this PR are old. Like this one, it should have an es6 extension. I'll go through and clean up the remote branch.

Reply:
Added some error handling for timeout (value set to 10 seconds, too long?) and for a returned status code of not 200.